scripts/common/setup_env.sh

```diff
@@ -45,24 +45,27 @@ else
 fi
 
 # Update IPDK RECIPE libraries
-export LD_LIBRARY_PATH=$IPDK_RECIPE/install/lib:$IPDK_RECIPE/install/lib64:$LD_LIBRARY_PATH
-export PATH=$IPDK_RECIPE/install/bin:$IPDK_RECIPE/install/sbin:$PATH
+export LD_LIBRARY_PATH=$P4CP_INSTALL/lib:$P4CP_INSTALL/lib64:$LD_LIBRARY_PATH
```

Comment:
if user chose default INSTALL path, he will not know that P4CP_INSTALL is same as IPDK_RECIPE/install.
How do we tell user that default install path is $IPDK_RECIPE/install and we need to use that here?

Reply:
These files will be used for setting the runtime environment. We will only package the binaries and libraries for user and not the source code. So IPDK_RECIPE/install doesn't make sense. I have added explanation of arguments to the scripts and running_infrap4d readme will specify the default location of all P4 component binaries on ACC once tarball is extracted